### ISSUE TITLE
Make `data-transfer` folder configurable

### ISSUE THREAD

**Issue by gschmutz** [TrivadisPF/modern-data-platform-stack#50]
Currently the `data-transfer` folder is mapped into some of the services (where it makes sense). This should be configurable, so that it can be disabled.

**Comment by ufasoli** [comment]
I think we could also make the path overridable (at least the mapping on the host side)

**Comment by gschmutz** [comment]
Yes, that is a good idea and I had that in mind a while ago as well. So instead of having `container_volume` static, make it configurable with this value as the default?

**Comment by ufasoli** [comment]
yes exactly have : 
./data-transfer:/tmp/data-transfer
as default value and be able to override the left part like so :
{{local_path}}:/tmp/data-transfer

**Comment by gschmutz** [comment]
just realized that you are talking about the `data-transfer` but yes, I still agree. 

could you also make sure that it is always mapped into the same folder inside the different containers. Most of the mappings are 

```
    - ./data-transfer:/data-transfer
```

and I think that is better than `/tmp/data-transfer`
